src/v2/Components/__tests__/SaleDetailTimer.jest.tsx

```diff
@@ -0,0 +1,116 @@
+import { getTimerCopy } from "../SaleDetailTimer"
+import moment from "moment"
+
+jest.useFakeTimers()
+
+jest.mock("v2/Utils/getCurrentTimeAsIsoString")
+
+require("v2/Utils/getCurrentTimeAsIsoString").__setCurrentTime(
+  "2022-03-21T20:22:32.000Z"
+)
+
+describe("getTimerCopy", () => {
+  describe("when the sale is open", () => {
+    describe("when the close date/time is more than 24 hrs before closing", () => {
+      const time = { days: "1", hours: "4", minutes: "00", seconds: "00" }
+      const startAt = moment().subtract(1, "hours")
```

Comment:
I find these a lot more readable if you feed it a specific date and not a moment calculated date. But maybe that's just me?